### PR TITLE
vc14 package - 2019-04-19 - 14.28.29914.0

### PR DESCRIPTION
**Pull request by derekgroh**
# Description

Matches the checksum for vc14 packages as they are updated on a static link, invalidating older configuration.

## Issues Resolved

None, same PR intent as https://github.com/sous-chefs/vcruntime/pull/39

## Check List

- [X] All tests pass. See TESTING.md for details.
- [X] New functionality includes testing.
- [X] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ This file is used to list changes made in each version of vcruntime.
 
 ## 2.2.0 - *2021-01-16*
 
+- update vc14 checksum for '14.28.29914.0' [@derekgroh](https://github.com/derekgroh)
```
Comment (ramereth) [inline CHANGELOG.md:9]: This needs to be moved up to the `# Unreleased` section.

File: metadata.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
 maintainer_email  'help@sous-chefs.org'
 license           'Apache-2.0'
 description       'Installs Microsoft Visual C++ Runtimes'
-version           '2.2.0'
+version           '2.2.1'
```
Comment (ramereth) [inline metadata.rb:6]: Please do not bump this, our automated tooling will do this when we merge it.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by derekgroh** [comment]
Not quite sure why the windows-2016 test loaded the old inspec test:

`         * windows_package[Microsoft Visual C++ 2019 Redistributable (x86) - 14.28.29914.0] action install`

``expected that `System Package Microsoft Visual C++ 2015-2019 Redistributable (x64) - 14.28.29325` is installed``

Should be good otherwise

**Comment by ramereth** [comment]
@derekgroh I hope you don't mind but I went ahead and rebased your PR so that the git history was a little cleaner.

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [2.2.1](https://github.com/sous-chefs/vcruntime/releases/tag/2.2.1)

**Pull request merged**